src/main/cpp/audioengine.cpp

```diff
@@ -271,12 +283,16 @@ namespace MWEngine {
 
     bool AudioEngine::render( int amountOfSamples )
     {
-        if ( thread == 0 )
-            return false;
-
         size_t i, j, k, c, ci;
         float sample;
 
+        if ( !threadOptimized ) {
+            if ( !DriverAdapter::isMocked() ) {
+                PerfUtility::optimizeThreadPerformance( AudioEngineProps::CPU_CORES );
+            }
```

Comment:
Thread optimization should be done in the render callback as AAudio's callback mechanism is executed on a different thread